conf/layer.conf

```diff
@@ -52,5 +52,8 @@ LICENSE_FLAGS_WHITELIST += "commercial_faad2"
 # Enable early start of Thunder
 DISTRO_FEATURES_append = " compositor thunder"
 
+#Enable this flag if the platform required FirmwareControl feature
+#DISTRO_FEATURES_append += " firmwarecontrol"
+
```

Comment:
We do not need to create a distro feature for firmware control, as it doesn't span across multiple plugins/software components.

This can be entirely controlled through the Package Config on WPEFramework Plugins. Suggest to remove distro feature usage.

Reply:
Thank you Wouter for the comments, will update